<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerVMAddDialog</class>
 <widget class="QDialog" name="ManagerVMAddDialog">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>541</width>
    <height>250</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Add a virtual machine</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="sizeConstraint">
    <enum>QLayout::SetFixedSize</enum>
   </property>
   <item row="7" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit_2"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0" colspan="2">
    <widget class="QLineEdit" name="lineEdit_3">
     <property name="enabled">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pushButtonBrowse">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Browse...</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Path:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Description:</string>
     </property>
    </widget>
   </item>
   <item row="11" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1" colspan="2">
    <widget class="QLineEdit" name="lineEdit"/>
   </item>
   <item row="11" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Cancel</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="2">
    <widget class="QLabel" name="labelPathDisplay">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="labelClone">
     <property name="text">
      <string>Virtual machine &quot;%1&quot; will be cloned into:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QCheckBox" name="checkBoxImport">
     <property name="text">
      <string>Import VM Files from:</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QCheckBox" name="checkBoxStartVM">
     <property name="text">
      <string>Start this virtual machine now</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1" colspan="2">
    <widget class="QCheckBox" name="checkBoxConfigureVM">
     <property name="text">
      <string>Configure this virtual machine now</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
